airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -2768,11 +2771,26 @@ def prepare_template(self) -> None:
             with open(self.configuration) as file:
                 self.configuration = json.loads(file.read())
 
+    def _add_job_labels(self) -> None:
+        dag_label = self.dag_id.lower()
+        task_label = self.task_id.lower()
+
+        if LABEL_REGEX.match(dag_label) and LABEL_REGEX.match(task_label):
+            automatic_labels = {"airflow-dag": dag_label, "airflow-task": task_label}
+            if "labels" in self.configuration:
+                if isinstance(self.configuration["labels"], dict):
```

Comment:
How about `isinstance(self.configuration.get("labels"), dict)`?

Reply:
Yep, this makes the conditions more explicit and concise, thanks!